SYCL/Regression/device_num.cpp

```diff
@@ -68,8 +69,9 @@ void PrintSystemConfiguration() {
     auto PlatformName = Platform.get_info<info::platform::name>();
     const auto &Devices = Platform.get_devices();
     for (const auto &Device : Devices) {
-      std::cout << Backend << ":" << getDeviceTypeName(Device) << ":"
-                << DeviceNums[Backend] << std::endl;
+      std::cout << backend_to_string(Backend) << ":"
```

Comment:
I would split this into a separate PR so that the current one only had `#include <iostream>` and could be committed without the main patch.